docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,791 @@
+depending on whether you authenticate as a local Teleport user or via the
+`github`, `saml`, or `oidc` authentication connectors: 
+
+<Tabs>
+<TabItem label="Local User">
+
+Retrieve your local user's configuration resource:
+
+```code
+$ tctl get users/$(tsh status -f json | jq -r '.active.username') > myuser.yaml
+```
+
+<Details title='Getting a "command not found" error?'>
+This command requires `jq` to be installed on your workstation, which you can do
+via the [`jq` download page](https://stedolan.github.io/jq/download/).
+</Details>
+
+Edit `myuser.yaml`, adding `teleport-event-handler-impersonator` to the list of
+existing roles.
+
+Apply your changes:
+
+```code
+$ tctl create -f myuser.yaml
+```
+</TabItem>
+<TabItem label="GitHub">
+
+Retrieve your `github`  configuration resource:
+
+```code
+$ tctl get github/github > github.yaml
+```
+
+Edit `github.yaml`, adding `teleport-event-handler-impersonator` to the
+`teams_to_roles` section. The team you will map to this role will depend on how
+you have designed your organization's RBAC, but it should be the smallest team
+possible within your organization. This team must also include your user.
+
+Here is an example:
+
+```diff
+  teams_to_roles:
+    - organization: octocats 
+      team: admins 
+      roles:
+        - access
++       - teleport-event-handler-impersonator
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f github.yaml
+```
+</TabItem>
+<TabItem label="SAML">
+
+Retrieve your `saml`  configuration resource:
+
+```code
+$ tctl get saml/mysaml > saml.yaml
+```
+
+Edit `saml.yaml`, adding `teleport-event-handler-impersonator` to the
+`attributes_to_roles` section. The attribute you will map to this role will
+depend on how you have designed your organization's RBAC, but it should be the
+smallest group possible within your organization. This group must also include
+your user.
+
+Here is an example:
+
+```diff
+  attributes_to_roles:
+    - name: "groups" 
+      value: "my-group" 
+      roles:
+        - access
++       - teleport-event-handler-impersonator
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f saml.yaml
+```
+</TabItem>
+<TabItem label="OIDC">
+
+Retrieve your `oidc`  configuration resource:
+
+```code
+$ tctl get oidc/myoidc > oidc.yaml
+```
+
+Edit `oidc.yaml`, adding `teleport-event-handler-impersonator` to the
+`claims_to_roles` section. The claim you will map to this role will depend on
+how you have designed your organization's RBAC, but it should be the smallest
+group possible within your organization. This group must also include your
+user.
+
+Here is an example:
+
+```diff
+  claims_to_roles:
+    - name: "groups" 
+      value: "my-group" 
+      roles:
+        - access
++       - teleport-event-handler-impersonator
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f saml.yaml
+```
+</TabItem>
+</Tabs>
+
+Log out of your Teleport cluster and log in again to assume the new role.
+
+### Export the access plugin identity
```

Comment:
I dont see any -ttl flags on the auth commands, or any mention or reloading them later.  Wouldnt this setup break after twelve hours?  Should we encouraged longer lived certs?  Machine ID might be an option here too

Reply:
I've added the `--ttl` flag to the `tctl auth sign` command snippets and mentioned Machine ID in the "Next steps" section (as well as the "Next steps" section in the Elastic and Fluentd guides). Eventually, I want to change all of our plugin instructions so they use Machine ID: https://github.com/gravitational/teleport/issues/15062